### DISCUSSION TITLE
[ FF7 ] cant open the game, gives me a crash error when i try to enter

### DISCUSSION THREAD

**Discussion by Kamikaze1903** [julianxhokaxhiu/FFNx#179] [General]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5553067/FFNx.log)

[crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/5553077/crash.zip)

**Comment by eve-atum** [comment]
Turn off the 60fps option in the animations mod.

**Comment by julianxhokaxhiu** [comment]
As said above, please if you use 7th reach to their related support channels. Thank you in advance.
